src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -13,15 +13,61 @@ public class HelpCommand extends Command {
             + "Example: " + COMMAND_WORD;
 
     public static final String SHOWING_HELP_MESSAGE = "Command List: add, delete, edit, find, list\n"
-            + "Example for Add command: `add n/[NAME] th/[TELEGRAM HANDLE] e/[EMAIL] s/[STUDENT_STATUS] r/[ROLE]`\n"
-            + "Example for Delete command: `delete [INDEX]` OR `delete [NAME]`\n"
-            + "Example for Edit command: `edit [INDEX] [PREFIX] [new description]`\n"
-            + "Example for Find command: `find [KEYWORD]`\n"
-            + "Example for List command: `list` OR `list all` OR `list contacts` OR `list all contacts`\n"
-            + "For more details, visit our user guide on the popup screen\n";
+            + "For more details, type `help [COMMAND_WORD]` with the words found in the command list above\n"
```

Comment:
Should we include commands `clear` and `exit`? (I vaguely recall there is `exit` command though correct me if I am wrong.